src/remote.jsx

```diff
@@ -83,8 +84,12 @@ function initializeEditor( assetsResult ) {
 
 function handleError( err ) {
 	error( 'Error initializing editor', err );
-	// Fallback to the local editor and display a notice. Because the remote
-	// editor loading failed, it is more practical to rely upon the local
-	// editor's scripts and styles for displaying the notice.
-	window.location.href = 'index.html?error=gbkit_global_unavailable';
+	if ( isDevMode() ) {
+		warn( 'Dev mode disabled automatic redirect to the local editor.' );
+	} else {
+		// Fallback to the local editor and display a notice. Because the remote
+		// editor loading failed, it is more practical to rely upon the local
+		// editor's scripts and styles for displaying the notice.
+		window.location.href = 'index.html?error=gbkit_global_unavailable';
+	}
```

Comment:
This automatic redirect, and others like it, made it difficult to debug failures occurring when loading the remote editor. The new dev mode allows disable automatic redirects so one can inspect any errors.